### ISSUE TITLE
PokeBuddy Transferring Pokémons That Were Not Supposed To Be Transferred

### ISSUE THREAD

**Issue by CapBlackShot** [finndev/PokeBuddy#1321]
Hi,

PokeBuddy seems to be transferring the wrong Pokémons. For example, 'Pidgey' is marked on the 'Do not transfer' list, but all of them (except the strongest one) are being transferred away. I'll leave my settings file and log down below.

Settings: http://pastebin.com/TE1zX4M5
Log: http://pastebin.com/jePKyfD6

Thanks,
BlackShot

**Comment by MukyGamesNL** [comment]
got the same problem with my Tauros

**Comment by CapBlackShot** [comment]
Yeah, this option doesn't seem to be working very well... let's hope they can see it and fix it. :)
